.pre-commit-config.yaml

```diff
@@ -1,4 +1,11 @@
 repos:
+-   repo: local
+    hooks:
+    -   id: build-workflows
+        name: Validate github action workflows
+        entry: ./tests/build-workflows.py -f True
```

Comment:
does this run in CI as well, to prevent PRs that fail to run `./build-workflows.py` when they would need to?

Reply:
Yes see https://github.com/Chia-Network/chia-blockchain/pull/8728/checks?check_run_id=3805955316#step:4:70